templates/etc/icinga2/constants.conf.j2

```diff
@@ -9,17 +9,17 @@
  */
 
 /* The directory which contains the plugins from the Monitoring Plugins project. */
-const PluginDir = "/usr/lib64/nagios/plugins"
+const PluginDir = "{{ icinga2_master_nagios_plugin_dir }}"
```

Comment:
`icinga2_master_nagios_plugin_dir` defaults to `/usr/lib/nagios/plugins` and not `/usr/lib64/nagios/plugins`. Intended?

Reply:
Yes, as the role now targets Debian 12 instead of RH, where this is the default.

Reply:
In that case, I think we should make the default based on the `ansible_os_family`. 
```suggestion
const PluginDir = "{{ icinga2_master_nagios_plugin_dir | default(_icinga2_master_nagios_plugin_dir) }}"
```
Remove `icinga2_master_nagios_plugin_dir` from the defaults and add `_icinga2_master_nagios_plugin_dir` to the os-specific vars. This way we can make it configurable, but the default works for multiple os families.

Reply:
We decided on deprecating RHEL support; as there are no OS specific vars yet like in other roles, I'm for just keeping it like this.

For roles that support multiple distros, we usually do something like this and include them based on the OS:
https://github.com/adfinis/ansible-role-icinga2_web/tree/master/vars